Sources/Swifter/DemoServer.swift

```diff
@@ -17,7 +17,12 @@ public func demoServer(publicDir: String?) -> HttpServer {
 
     server["/"] = { r in
         var listPage = "Available services:<br><ul>"
-        listPage += server.routes.map({ "<li><a href=\"\($0)\">\($0)</a></li>"}).joinWithSeparator("")
+		for route in server.routes {
+			if route.method == .GET || route.method == .ANY {
+				listPage += "<li><a href=\"\(route.path)\">\(route.path)</a></li>"
+			}
+		}
```

Comment:
The issue with that becomes duplicated entries for paths that have multiple handlers for different methods (such as /login in this case). I agree that the map method is nicer, and one way to keep it it to prefix each link with the HTTP request method, but I thought it was better to instead only display the links that can be handled with GET requests (since they are clickable links which will produce GET requests after all).